docs/04_web/03_design-systems/03_preview.md

```diff
@@ -9,7 +9,72 @@ tags:
   - preview
 ---
 
+## Introduction
+
+The Design System Configurator you see below, will be the tool we use to configure the design tokens and determine the look that's best suited for our application.
+
+On the left hand side, is the [Editor](../../../web/design-systems/preview/#editor), where we can make all the modifications we want. On the right side, we can see the [Preview](../../../web/design-systems/preview/#preview) with the results of those changes.
+
+Try changing the configurations and observe the reflected changes on the right.
+
 <design-system-editor>
   <design-system-preview></design-system-preview>
   <design-system-export slot="export"></design-system-export>
-</design-system-editor>
\ No newline at end of file
+</design-system-editor>
+
+## Editor
+
+The checkbox in the editor is ticked by default.
+
+![](/img/checkbox.PNG)
+
+This means that when you switch between Design Systems, all of your actively made selections are retained. For instance, if you configure the design tokens in the `Zero` Design System, and then switch to the `Foundation UI`, you will see the changes you configured there as well. 
+
+Alternatively, if it's unticked, switching between Design Systems will not retain the configurations but only display the default values of the Design System selected.
+
+:::important
+An important configuration to note, is the `Design System`. In the dropdown list of `Design System` you will see `Zero` and `Foundation UI`. Depending on which one you choose to create your designs on, will give you a different look.
+:::
+
+In the editor, after we have made all the configurations to the design tokens, there are two more things we can do:
```

Comment:
It's true that we've configured design tokens, but we don't need to keep mentioning them - we can just say configured design system etc.